buildpack/telemetry/appdynamics_telegraf_output.py

```diff
@@ -0,0 +1,66 @@
+"""The module is used in runtime to convert metrics from telegraf for appdynamics Machine Agent"""
+import json
+import logging
+import requests
+from requests.exceptions import ConnectionError
+
+
+METRIC_TREE_PATH = "Custom Metrics|mxapp"
```

Comment:
How about?
```suggestion
METRIC_TREE_PATH = "Mx Runtime Statistics|app_environment_id"
```
So if the customer has multiple app environments, it will all show up under the `Mx Runtime Statistics` tree with the corresponding environment ids.